config/settings.yml

```diff
@@ -0,0 +1,6 @@
+links:
+  rails_tutorial:
+    home_page: http://www.railstutorial.org/
+    help_page: http://www.railstutorial.org/help
+    help_book: http://www.railstutorial.org/book
+    
```

Comment:
add endline for this file

Reply:
![image](https://user-images.githubusercontent.com/68467813/90376384-f5baa980-e0a8-11ea-908c-f2dc1b9ad886.png)

When i add endline, it become like this... no : found

Reply:
You can remove `end` and check again